src/NativeRopeContext.cpp

```diff
@@ -144,6 +145,25 @@ void NativeRopeContext::_simulate_velocities(double delta)
     std::swap(oldpoints, points);
 }
 
+void NativeRopeContext::_simulate_wind(PackedVector2Array* velocities) const
+{
+    if (!wind_enabled || wind.is_null() || wind->noise.is_null())
+        return;
+
+    const float time = (float)Time::get_singleton()->get_ticks_msec();
+
+    for (int i = 0; i < points.size(); ++i)
+    {
+        const float noise = wind->noise->get_noise_3d(points[i].x, points[i].y, time);
```

Comment:
That's a good idea! I first thought about projecting the point position vector on the wind direction but using  `get_noise_3d` is much simpler and straight forward.